tests/IntegrationTests/ItemPropertiesSetupTest.php

```diff
@@ -0,0 +1,104 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\IntegrationTests;
+
+use Lmc\Matej\Model\Command;
+
+/**
+ * @covers ItemPropertiesSetupRequestBuilder
+ * @covers EventsRequestBuilder
+ */
+class ItemPropertiesSetupTest extends IntegrationTestCase
+{
+    /** @test This throws an error in Matej's MGW, but API will return OK */
+    public function issueEventWithUnsetProperties(): void
```

Comment:
We use BDD-like naming of the test methods, with "should" as the first part of the test name, to make clear what behavior the object under test *should* have.

ItemPropertiesSetup
... should[ExpectedBehaviorOfTheObjectUnderTest]
... shouldCreateNewItemProperties
... shouldDeleteItemProperties
... etc